guides/common/modules/proc_adding-rhel-system-roles.adoc

```diff
@@ -1,5 +1,5 @@
 [id="adding-rhel-system-roles_{context}"]
-= Adding {RHEL} System Roles
+= Adding {RHEL} system roles
```

Comment:
Is this the name of a product? See line 4.

Reply:
good catch!